docs/features/files.md

```diff
@@ -1,5 +1,14 @@
 # Files and volumes
 
+## File mapping
+
+It is possible to map a file or directory from your FileSystem into the container as a volume using `withFileSystemBind`:
+```java
+String pathToFile = String.format("%s/.aws", System.getProperty("user.home"))
```

Comment:
For conciseness I'd suggest just doing:
```suggestion
String pathToFile = System.getProperty("user.home") + "/.aws";
```

As it's a small change I'll apply and merge.